Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -50,13 +50,12 @@ GameFile::GameFile(const QString& path) : m_path(path)
 
   if (!TryLoadCache())
```

Comment:
it could, but i don't think (ab)using c/c++ `||`-short-circuiting like this is really good for readability.